### PR TITLE
Fix api limiter

### PR DESCRIPTION
**Pull request by DisneyManBro**
Fixed the following:
- replenishTokens() was updating lastTokenReplenish even when no tokens were replenished
- WOMClient is an object of clients so its Proxy was returning another client instead of a proxy of that client with the same ratelimiter

### CODE REVIEW

File: config/wom.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 const { WOMClient } = require('@wise-old-man/utils');
 const { Environment } = require('../services/environment');
 
-const WOM_RATE_LIMIT = 100;
+const WOM_RATE_LIMIT = Environment.WOM_API_KEY ? 100 : 20;
+const TIME_PER_TOKEN = 60000 / WOM_RATE_LIMIT;
```
Comment (Kojonkler) [inline config/wom.js:5]: I would rename this slightly to make explicit that this is time expressed in milliseconds, so something like `TIME_PER_TOKEN_IN_MS` or `TIME_PER_TOKEN_MS` would suffice

File: config/wom.js [hunk 2]
```diff
@@ -40,16 +40,21 @@ const initialize = async () => {
     });
 };
 
+const handler = {
+    get(target, key) {
+        if (typeof target[key] === 'function') {
+            return (...args) =>
+                requestWithRateLimit(() => target[key](...args));
+        } else if (typeof target[key] === 'object' && target[key] !== null) {
```
[the remark this answers is not in the document]
Comment (DisneyManBro) [reply config/wom.js]: 6a5179c

File: config/wom.js (hunk 1 above)
Comment (DisneyManBro) [reply config/wom.js]: Changed in 6a5197c

**Review by tomking — APPROVED** [review]
Let's get this merged in to test, huge thanks to @DisneyManBro

**Pull request merged**